packages/cli/src/lib/bundler/transforms.ts

```diff
@@ -16,14 +16,16 @@
 
 import webpack, { Module, Plugin } from 'webpack';
 import MiniCssExtractPlugin from 'mini-css-extract-plugin';
-import { BundlingOptions } from './types';
+import { BundlingOptions, BackendBundlingOptions } from './types';
 
 type Transforms = {
   loaders: Module['rules'];
   plugins: Plugin[];
 };
 
-export const transforms = (options: BundlingOptions): Transforms => {
+export const transforms = (
+  options: BundlingOptions | BackendBundlingOptions,
```

Comment:
Could probably just change this to only needing isDev in options for now. Also thinking we might want to split out backend transforms so we're loading loading css files and whatnot.